### PR TITLE
33 payment type view

### PR DESCRIPTION
**Pull request by Anulfo**
## Status

IN DEVELOPMENT
## Documentation

[x ] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

NO
## Description

It's a form with 2 fields two retrieve the `string` for Payment Type and the `string` for Account Number.
In the controller `PaymentTypeController` there is an async method  `Create` that POST the `PaymentType` object to the database.  
## Related PRs

branch | PR
## Resolves Issue Number

number [link]()
#33
## Deploy Notes

Dotnet Run
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.

``` sh
git pull --prune
git checkout <33-paymentTypeView>
dotnet run
```

I think there is no need for migrations stuff but there could be mines.
## Impacted Areas in Application

Just Payment Type View and a little of Cart view, trying to attach a button there that takes to the creation of a new payment type

### CODE REVIEW

**Comment by garrett-vangilder** [comment]
LGTM 

Only thing that could be change is the button from the cart should route the user to the form, unfortunately I did not see that functionality implemented. This could be a future ticket in my opinion.

**Comment by daynewright** [comment]
LGTM

**Review by daynewright — APPROVED** [review]
LGTM

**Comment by garrett-vangilder** [comment]
LGTM

File: Controllers/PaymentTypesController.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        return View();
+        }
+
+        [HttpPost]
+
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create(PaymentType paymentType)
+        {
+            paymentType.UserId = 1;
+            if (ModelState.IsValid)
```
Comment (MattKraatz) [inline Controllers/PaymentTypesController.cs:41]: @daynewright I can't understand how ModelState is valid here but not when we were adding the user to that product edit. Good stuff!

**Review by MattKraatz — APPROVED** [review]
LGTM

**Pull request merged**